clients/drcachesim/scheduler/scheduler.cpp

```diff
@@ -1701,6 +1701,24 @@ scheduler_tmpl_t<RecordType, ReaderType>::get_initial_input_content(
     // output stream(s).
     for (size_t i = 0; i < inputs_.size(); ++i) {
         input_info_t &input = inputs_[i];
+
+        // If the input jumps to the middle immediately, do that now so we'll have
+        // the proper start timestamp.
+        if (!input.regions_of_interest.empty() && options_.mapping != MAP_AS_PREVIOUSLY) {
```

Comment:
Is there a valid config where `!input.regions_of_interest.empty()` and `options_.mapping == MAP_AS_PREVIOUSLY`?

IOW: would the user specify region of interests and want to replay a prior schedule? I guess maybe. But what if the prior schedule skipped over the presently specified region of interest?

Reply:
The docs say that for replay the ROI are ignored:
```
         * The same output count and input stream order and count must be re-specified;
         * scheduling details such as regions of interest and core bindings do not
         * need to be re-specified and are in fact ignored.
```
Maybe it should be an error to pass them?  Some of our unit tests pass them today.

Reply:
I added a comment about this.